clarity.d.ts

```diff
@@ -4,6 +4,7 @@
 
 export function start(config?: IConfig): void;
 export function stop(): void;
+export function trigger(key: string): void;
```

Comment:
it's a small change, but this merits a version bump imo (so we can know which version you're referencing will contain something that can be relied on)

Reply:
The will be a version change coming from a second PR. No need to split them in two version bumps because they will be deployed together